packages/backend/server/src/plugins/copilot/providers/types.ts

```diff
@@ -144,20 +144,20 @@ export interface CopilotTextToImageProvider extends CopilotProvider {
 export interface CopilotImageToTextProvider extends CopilotProvider {
   generateText(
     messages: PromptMessage[],
-    model: string,
+    model?: string,
     options?: CopilotChatOptions
   ): Promise<string>;
   generateTextStream(
     messages: PromptMessage[],
-    model: string,
+    model?: string,
     options?: CopilotChatOptions
   ): AsyncIterable<string>;
 }
 
 export interface CopilotImageToImageProvider extends CopilotProvider {
   generateImages(
     messages: PromptMessage[],
-    model: string,
+    model?: string,
```

Comment:
model should not be optional

Reply:
Why model in `generateImagesStream` is optional and model in `generateImages` not optional before? I think they shoud be the same type.

Reply:
then the type of `generateImagesStream` should be fix